package.json

```diff
@@ -51,6 +51,7 @@
     "eslint-config-prettier": "^8.5.0",
     "eslint-plugin-cypress": "^2.12.1",
     "eslint-plugin-storybook": "^0.5.7",
+    "genversion": "^3.1.1",
```

Comment:
Great solution. Didn't know of that yet. Great job researching the security implications 👍